### PR TITLE
V6.9.0 with v10.1.3

### PR DESCRIPTION
**Pull request by RitudhwajA**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ pre: "<b>6. </b>"
 math: true
 ---
 
-### [6.9.0] - UNRELEASED
+### [6.9.0] - 2024-11-08
```
Comment (KtorZ) [inline CHANGELOG.md:10]: Good point 😅

File: .github/workflows/continuous-integration.yaml [hunk 2]
```diff
@@ -402,10 +402,10 @@ jobs:
     strategy:
       matrix:
         os: [ linux ]
-        target: [ cardano-node-ogmios ]
+        target: [ ogmios ]
```
Comment (KtorZ) [inline .github/workflows/continuous-integration.yaml:405]: Hmm, that's not going to work; the cardano-node is specifically needed for the `cardano-node-ogmios` target for the image.

Comment (RitudhwajA) [reply .github/workflows/continuous-integration.yaml:405]: sorry i forked the repo for creating image. Didn't realised i created a PR here :D

File: CHANGELOG.md (hunk 1 above)
Comment (RitudhwajA) [reply CHANGELOG.md:10]: haha yes

File: .github/workflows/continuous-integration.yaml (hunk 2 above)
Comment (KtorZ) [reply .github/workflows/continuous-integration.yaml:405]: Well, I was assuming you might do one. I would happily merge it :)

Comment (RitudhwajA) [reply .github/workflows/continuous-integration.yaml:405]: I will do one in that case. Better for me as well.

Comment (RitudhwajA) [reply .github/workflows/continuous-integration.yaml:405]: Are you working on it already @KtorZ ? I see some recent changes related to v10.1.3

Comment (KtorZ) [reply .github/workflows/continuous-integration.yaml:405]: Indeed. Thought I might as well just do it given it's a few lines.

**Pull request closed without merging**